support-data-rest/src/main/java/org/uniprot/api/support/data/configure/domain/AdvancedSearchTerm.java

```diff
@@ -108,7 +107,11 @@ private static AdvancedSearchTerm from(SearchFieldItem fi) {
         if (fi.getFieldType() != null) {
             b.fieldType(fi.getFieldType().name().toLowerCase());
             if (fi.getFieldType() == SearchFieldType.EVIDENCE) {
-                b.evidenceGroups(AnnotationEvidences.INSTANCE.getEvidences());
+                if (fi.getId().equalsIgnoreCase("go_evidence")) {
+                    b.evidenceGroups(GoEvidences.INSTANCE.getEvidences());
+                } else {
```

Comment:
it would be better if we put comment that go_evidence has different kind of evidences